docs/docs/deployment/overview.md

```diff
@@ -179,31 +179,56 @@ the platform. For example, if you wanted to disable Google OAuth authentication,
+| `dark_mode`                           | Enables Dark Mode in UI See Below                                                                                                               | None                                                                                                         |
+| `demo_feature`                        | Shows a demo feature in the side navigation bar!                                                                                                | green                                                                                                        |
+| `disable_create_org`                  | Turning this on will prevent users from creating any additional organisations                                                                   | None                                                                                                         |
+| `disable_users_as_reviewers`          | If enabled, this flag will hide the Assigned users section in the Change Requests and in the Create Change Request modal in the Features page.  | None                                                                                                         |
+| `edge_identities`                     | Uses the new edge api for identities if the project is setup                                                                                    | None                                                                                                         |
+| `enable_groups_as_reviewers`          | If enabled, this flag will show the Assigned groups section in the Change Requests and in the Create Change Request modal in the Features page. | None                                                                                                         |
+| `enable_metadata`                     | If enabled, metadata can be handled                                                                                                             | None                                                                                                         |
+| `feature_name_regex`                  | Enables the project setting to add a regex matcher to validate feature names                                                                    | None                                                                                                         |
+| `feature_versioning`                  | Opt into feature versioning for your environment                                                                                                | None                                                                                                         |
+| `flag_analytics`                      | Flag usage chart - requires additional infrastructure (See here)                                                                                | None                                                                                                         |
```

Comment:
Looks like we've lost the link on this one? `(See _here_)` should link to (/deployment/overview#flag-analytics)